docs/FREE_TIER.md

```
@@ -0,0 +1,74 @@
+# Career-Ops on the Free Tier (Gemini CLI)
+
+career-ops works with **Gemini CLI's free tier** — no API key or paid
+subscription required. This guide covers setup, limits, and trade-offs.
+
+## Quick Start
+
+1. Install Gemini CLI (requires Node.js 18+):
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)